### ISSUE TITLE
Enhancement：Provides zoom in Live logcat mode

### ISSUE THREAD

**Issue by Shawnli3287** [Tortel/SysLog#30]

**Comment by RokeJulianLockhart** [comment]
> https://github.com/Tortel/SysLog/issues/30#issue-1194026023

@Shawnli3287, surely merely adhering to the system font (size) preference is superior? No alternative text field provides zoom functionality - that's more the purview of an accessibility toolset.
